### DISCUSSION TITLE
Thank you. Works like a charm!

### DISCUSSION THREAD

**Discussion by chris-numeus** [AdnanHodzic/displaylink-debian#934] [General]
Works on Debian 12. Thank you for creating this!

```log
--------------- Linux system info ----------------

Distro: Debian
Release: bookworm
Kernel: 6.1.0-21-amd64

---------------- DisplayLink info ----------------

Driver version: 1.13.1
1.14.4
DisplayLink service status: up and running
EVDI service version: 1.14.4

```

**Comment by HanSyt** [comment]
Why using this? At synaptic you can find the repository data for Ubuntu, works good and does automatically update your driver.
https://www.synaptics.com/products/displaylink-graphics/downloads/ubuntu

Tested it With Debian, KDE Neon, Linux Mint.

**Reply by mrdotballard** [reply]
Been spending a few hours on this, and it seems the edvi version that ships with the ubuntu drivers is not compatible with the Debian 12 kernal that i've got. I tried to get edvi 1.12.0 installed on debian but no joy. Would you mind sharing what debian kernal you tested it on, and the displaylink drivers worked straight out of the box??

**Comment by HanSyt** [comment]
![Screenshot_20241005_151329](https://github.com/user-attachments/assets/0174b28b-615f-4e75-b842-f7687bbbc501)

Version of EVDI 1.14.6-100 is causing problems in debian, switch back to a lower version and hold updates. As you can see there is a newer version, don' t know if the newer one is working.

Linux Latitude5400 6.1.0-26-amd64 #1 SMP PREEMPT_DYNAMIC Debian 6.1.112-1 (2024-09-30) x86_64 GNU/Linux

**Comment by Marakai** [comment]
Just adding to the praise! I've been slamming my head against a wall with the default DisplayLink drivers for my Startech adapter and 2 WiMaxit touch screens.

Installed yours, rebooted and glory be!

Kubuntu 24.04
